manifests/params.pp

```diff
@@ -1,7 +1,7 @@
 class solr5::params {
     $package_url = 'http://ftp.halifax.rwth-aachen.de/apache/lucene/solr/5.2.1/solr-5.2.1.tgz'
     $package_version = '5.2.1'
-    $package_target_dir = '/usr/local/src'
+    $package_target_dir = '/tmp'
```

Comment:
To unpack everything under /tmp, the software is installed later under /opt. I do not see a reason to keep the archive explicitly.